### PR TITLE
Jon/93 bintray

### LINKED ISSUE #71: generate RPMs
We need to generate RPMs if we're going to do releases.  We already generate DEB packages with FPM, we should stick with that system.

This should hook into gradle's build system the same way generating deb packages do.

### LINKED ISSUE #93: publish to bintray

### PR DESCRIPTION
**Pull request by rustyrazorblade**
Reworked gradle config to be ready for 1.0

    * Using gradle for all things now, including builds and uploads
    * cleaned up the gradle build script, removed unused / deprecated functionality
    * switched from FPM to Netflix's Nebula deb & RPM build tool (ospackage)
    * switched from my trash home brew uber jar code to the shadow plugin
    * correctly using the application plugin, ties into Nebula (ospackage_application)
    * fixed build order dependency stuff

    Closes #93
    Closes #71

### CODE REVIEW

File: manual/MANUAL.adoc [hunk 1]
```diff
@@ -29,17 +29,64 @@ automatically captured by the Dropwizard metrics library.
...
+==== Deb Packages
+
+```
+echo "deb https://dl.bintray.com/thelastpickle/tlp-tools-deb weezy main" | sudo tee -a /etc/apt/sources.list
+apt install tlp-stress
+```
+
+==== RPM Packages
+
+You'll need the bintray repo set up on your machine.  Create this `/etc/yum.repos.d/tlp-tools.repo `:
```
Comment (ossarga) [inline manual/MANUAL.adoc]: Typo: remove space in "`/etc/yum.repos.d/tlp-tools.repo `". Change to "`/etc/yum.repos.d/tlp-tools.repo`"

File: build.gradle [hunk 2]
```diff
@@ -118,17 +111,90 @@ test {
...
+    dependsOn("generateExamples")
+
+    environment "TLP_STRESS_VERSION", "${version}"
     commandLine 'docker-compose', 'up', 'docs'
-    group = 'Build'
+    group = "Documentation"
     description = "Build website documentation"
+}
+
+task generateExamples(type: Exec) {
```
[the remark this answers is not in the document]
Comment (rustyrazorblade) [reply build.gradle]: Good idea.

File: manual/generate_examples.sh [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
```
[the remark this answers is not in the document]
Comment (rustyrazorblade) [reply manual/generate_examples.sh]: Can't hurt, I'll add it.

File: manual/MANUAL.adoc [hunk 4]
```diff
@@ -29,17 +29,64 @@ automatically captured by the Dropwizard metrics library.
...
+
+The easiest way to get started is to use your favorite package manager.
+
+The current version is {TLP_STRESS_VERSION}.
+
+==== Deb Packages
+
+```
+echo "deb https://dl.bintray.com/thelastpickle/tlp-tools-deb weezy main" | sudo tee -a /etc/apt/sources.list
+apt install tlp-stress
```
[the remark this answers is not in the document]
Comment (rustyrazorblade) [reply manual/MANUAL.adoc]: That's weird, I thought I had set everything up to be signed.  I'll go through each of the repos and double check.

[the remark this answers is not in the document]
Comment (rustyrazorblade) [reply manual/MANUAL.adoc]: OK, I think I've fixed this.  I tweaked a couple settings on the repo and modified the instructions per your feedback.  I've also added the step to do the following:
`sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 2895100917357435` to get the TLP we're using to sign this repo.

[the remark this answers is not in the document]
Comment (ossarga) [reply manual/MANUAL.adoc]: As discussed we will still need to do an `apt update` before the install. This is so the local `apt` db is updated.

**Pull request merged**